templates/charms-architecture.html

```diff
@@ -0,0 +1,191 @@
+{% extends 'how-juju-works.html' %}
+
+{% block title %}Charms architecture{% endblock %}
+{% block meta_keywords %}Software operators{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1nMN5UlwGqI4C4GmYcKy4yRzd9ANUmsVyB1ZcKY7Wpro/edit#heading=h.4tjfso5hx6tw{% endblock meta_copydoc %}
+{% block page_class %}is-paper{% endblock %}
+
+{% set selected_tab='charms-architecture' %}
+
+{% block how_juju_works_content %}
+<section class="p-strip u-no-padding--bottom u-no-padding--top">
```

Comment:
The only thing strip does (`p-strip`) is adding top and bottom padding. If padding is removed with `u-no-padding--top` and `u-no-padding--bottom` the strip is no longer needed. So all these class names could be removed (or possibly whole section could be removed (although having `<section>` may have a value from semantic point of view to group markup).